demo-angular/package.json

```diff
@@ -35,7 +35,9 @@
     "nativescript-css-loader": "~0.26.0",
     "nativescript-dev-typescript": "~0.7.0",
     "nativescript-dev-webpack": "~0.18.0",
-    "tslint": "~5.11.0"
+    "tslint": "~5.11.0",
+    "@angular/compiler-cli": "~7.1.0",
```

Comment:
The nativescript-dev-webpack plugin adds @angular/compiler-cli and @ngtools/webpack as devDependencies if they are not present, so we prefer to not include these in the package.json in order to avoid conflicting versions.